docs/native/src/differences.rst

```diff
@@ -1,3 +1,12 @@
+..  _differences:
+
+Differences between |Office iOS| and |wac|
+==========================================
+
+Unlike |wac|, partners need to implement all Container and Bootstrapper operations. You can find more information about these operations in the :ref:`WOPI implementation requirements for Office for iOS integration section. <requirements>`
+
+|wac| has the following WOPI implementation requirements are not needed for |Office iOS| integration: 
+ * :ref:`Proof keys <Proof Keys>`
+ * :ref:`Breadcrumbs <breadcrumbs>`
+ * File Operations: :ref:`UnlockAndRelock <UnlockAndRelock>`, :ref:`PutRelativeFile <PutRelativeFile>`, :ref:`PutUserInfo <PutUserInfo>` 
```

Comment:
UnlockAndRelock must be implemented by all partners; it's part of the locks requirement.